PerimeterXModule/Internals/PxContext.cs

```diff
@@ -218,27 +218,19 @@ public PxContext(HttpContext context, PxModuleConfigurationSection pxConfigurati
 
         private bool shouldMonitorRequest(HttpContext context, PxModuleConfigurationSection pxConfiguration)
         {
-            string uri = context.Request.Url.AbsolutePath;
-            if (uri.IndexOf("/", StringComparison.Ordinal) == 0)
-            {
-                uri = uri.Substring(1);
-            }
+            string uri = context.Request.Url.AbsolutePath.TrimStart('/');
 
-            var mitigationUrls = pxConfiguration.MitigationUrls;
-            if (mitigationUrls.Count > 0)
-            {
-                return !mitigationUrls.Contains(uri);
-            }
+            bool isMitigationRoute = pxConfiguration.MitigationUrls?.Contains(uri) == true;
+            bool isMonitoredRoute = pxConfiguration.MonitoredRoutes?.Contains(uri) == true;
```

Comment:
Why do we need the `== true` part?


```suggestion
            bool isMitigationRoute = !!pxConfiguration.MitigationUrls?.Contains(uri);
            bool isMonitoredRoute = !!pxConfiguration.MonitoredRoutes?.Contains(uri);
```

Reply:
What you suggested is JavaScript syntax, not C#.
In any case, I’ve updated it to a clearer implementation.